src/modules/ce/components/referral/ReferralStepAction.tsx

```diff
@@ -56,21 +52,40 @@ const StartCeReferralStepButton: React.FC<Props> = ({
 
   if (error) throw error;
 
-  if (step.status !== CeReferralStepStatus.Available) return;
+  if (status === CeReferralStepStatus.Available && step.access.canPerformStep) {
+    return (
+      <LoadingButton
```

Comment:
nit: in the mocks these buttons have a fixed width (166px) that is wider and allows all buttons to have uniform width. I think that looks nice